### ISSUE TITLE
Map Accessibility: Label Contrast Ratio

### ISSUE THREAD

**Issue by TheMapSmith** [GreenInfo-Network/nj-cancer-maps#34]
## Text over an image lacks 4.5:1 contrast ratio

<img width="1919" height="894" alt="Image" src="https://github.com/user-attachments/assets/df8f5f06-a761-43a3-aadf-8ae369166bf0" />

All or portions of text over an image do not meet the minimum 4.5:1 contrast requirement. Small text and images of small text (under 18 point regular font or 14 point bold font) MUST have a contrast ratio of at least 4.5 to 1 with the background.

Element Description: The following elements
- 'NEW JERSEY' text on map

Description of the issue: 
The above-mentioned element lacks the minimum required color contrast ratio of 4.5:1 with its adjacent background color.

Contrast Details:
Text color on the map: #ADB8BF
Background color: #FFFFFF
Contrast Ratio: 2.02: 1 

How To Fix:
Fix this issue by adjusting the text and/or background to increase the contrast to at least 4.5 to 1. Common techniques include:
1. Applying an opaque or semi-opaque background behind the text.
2. Lighten or darken part or all of the image to increase the overall contrast between the text and the image.
3. Use a border around the letters to create sufficient contrast.

Reference:
Deque University: https://dequeuniversity.com/class/visual-design/contrast/text-against-background

----

Seeing as we can't adjust Positron Labels, this can't be fixed unless we create custom point labels in Leaflet
